src/vue/forms/InvestForm.vue

```diff
@@ -0,0 +1,564 @@
+<template>
+  <div class="invest-form">
+    <template v-if="isLoaded && quoteAssetBalances.length">
+      <form
+        novalidate
+        class="app__form"
+        @submit.prevent="isFormValid() && showConfirmation()"
+      >
+        <div class="app__form-row">
+          <div class="app__form-field">
+            <!--
+              :key is a hack to ensure that the component will be updated
+              after property change
+            -->
+            <select-field
+              :key="form.asset"
+              v-model="form.asset"
+              :values="quoteAssetListValues"
+              :label="'invest-form.asset-lbl' | globalize"
+              id="invest-asset"
+              @blur="touchField('form.asset')"
+              :disabled="formMixin.isDisabled || !canUpdateOffer"
+            />
+
+            <vue-markdown
```

Comment:
i'd rather suggest making `globalize` filter format markdown the same way the `vue-markdown` does. 
Using `vue-markdown` is confusing at first look and a bit dirty

@max-nes

Reply:
`globalize` cannot return html, but we need it here to format markdown

Reply:
unfortunately, filters cannot return anything but strings. markdown filter should returns html as plain text and we cannot create new DOM elements without directive or component